### PR TITLE
Initital Commit German Language (WIP)

### PR DESCRIPTION
**Pull request by gOOvER**

### CODE REVIEW

**Comment by pH-7** [comment]
Keep your good work! Well done @gOOvER 🎉

File: _protected/app/langs/de_DE/language.php [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+    /***** BYTE UNIT *****/
+    'byte' => '%0% Bytes',
+    'kilobyte' => '%0% KB',
+    'megabyte' => '%0% MB',
+    'gigabyte' => '%0% GB',
+    'terabyte' => '%0% TB',
+
+    /***** USER'S INFO FIELDS *****/
+    'address' => 'Adresse:',
+    'street' => 'Stasse:',
```
Comment (OttoKoester) [inline _protected/app/langs/de_DE/language.php:374]: 'street' => 'Stasse:', It's **'Strasse:'**
This translation has several erros and it's not German Translation for Germany Language, it looks like for Switzerland.

Comment (pH-7) [reply _protected/app/langs/de_DE/language.php:374]: Thanks for letting us know. Do you mind to fix those typos..?

Comment (gOOvER) [reply _protected/app/langs/de_DE/language.php:374]: @OttoKoester Do it better;) ist opensource and mb you don't know what WIP is.

So better learn the different Slangs and then answer. Your aswer is not really usefull.

**Pull request closed without merging**